module-services/service-time/AlarmOperations.cpp

```diff
@@ -1,11 +1,11 @@
-// Copyright (c) 2017-2021, Mudita Sp. z.o.o. All rights reserved.
+// Copyright (c) 2017-2023, Mudita Sp. z.o.o. All rights reserved.
 // For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
 
 #include "AlarmOperations.hpp"
 
 #include <module-db/Interface/AlarmEventRecord.hpp>
 #include <module-db/Interface/EventRecord.hpp>
-
+#include <EventStore.hpp>
```

Comment:
I know it's not your fault but `EventStore` is a really ugly way of achieving such a goal. It literally is a simple singleton in disguise :( Unfortunately, there is no other alternative for now that would not require us to introduce major changes.

Reply:
done